### ISSUE TITLE
Clarifications on remapping and uuids generated/used by wyng 

### ISSUE THREAD

**Issue by tlaurion** [tasket/wyng-backup#86]
As you know, when using multiple meta-dir to backup/receive/verify different subsets of archives, wyng remaps the volumes completely before doing a send. 

`Pairing snapshot` and Removed `mis-matched snapshot` happen in those `send --remap --clean` situations, which for TemplateVMs, root-autosnap and here again, StandaloneVMs (Windows) takes a really long time prior of actually doing the delta of those volumes.

The same situation is expected when doing a arch-init from when the user wants to restore a local LVM from a backup, but that is somewhat expected since the remote session can be a complete new volume altogether.

Is there any optimization that can be done on that? I remember we had a discussion on how volumes uuid were calculated and not reused between meta-dirs, which I think is still a problem. From my better and better understanding, the digests from meta-dir could be reused to calculate the differences here more efficiently, without flushing/renewing the uuid and consider it a whole different source LVM to compare it to?

**Comment by tasket** [comment]
Some ways to handle switching a volume between archives:

1. Remap, which re-scans/compresses all volume data (current solution)
2. Juggle multiple snapshots per volume so no re-scan is required
3. Add pre-compression hashes to metadata (probably never happen)
4. Specify the "other" archive that own's the snapshot in a sort of 3-body transaction.

The second option isn't so easy to do, as it multiplies the complexity of snapshot handling incl. cleanup issues.

The fourth option would involve Wyng updating archive 2 with data taken directly from archive 1 and then re-using the snapshot from archive 1. This skips any re-compression, since we are just matching existing hashes between the two. But there are some complexities like extra manifest processing, handling two archive meta dirs simultaneously, finding common points of reference between the two archives. This could also take the form of a command that updates one archive from a different archive, and then performing a `send` on top of that.

I think that archive-switching is a narrow corner case, and I noticed other "advanced" utilities also have to re-scan-compress (the difference being, unlike wyng, they do it often and IMO still end up w a big mess).  So probably the best near-term solution is to switch to a faster compression alg when it becomes available on your platform. Fedora 32 in Qubes 4.1 dom0, for example, makes it possible for me to support Zstandard there, which on a quad-core system delivers about 8X the throughput compared with bzip2 at similar compression ratios.

Underlining all this, of course, is that Wyng must make decisions about snapshot & archive state very cautiously. If not, it could create archives that verify OK but are internally corrupt.

---

TL;dr – You seem to be asking for some way to override UUID checking so a .tick snapshot can be re-used with a different archive. But I think what you really want is an archive-to-archive update or synchronization mechanism, which allows this kind of thing to be done cleanly. Until that happens, a faster compressor can cut remapping time by 7/8.
